File: java-concepts/src/main/java/org/demo/types/ValueTypes.java
package org.demo.types;

public class ValueTypes {

    boolean booleanVariable = true;
    short shortIntegral = Short.MAX_VALUE; // 32767
    int integral = Integer.MAX_VALUE; // 2147483647
    long longIntegral = Long.MAX_VALUE; // 9223372036854775807L
    float shortDecimal = Float.MAX_VALUE; // 3.4028235E38f
    double decimal = Double.MAX_VALUE; // 1.7976931348623157E308
    char singleCharacter = 'A';

    enum OrdinalEnum {
        E1, E2, E3
    }
}

File: csharp-concepts/types/ValueTypes.cs
namespace csharp_concepts.types;

public class ValueTypes
{
    bool booleanVariable = true;

    short shortIntegral = short.MaxValue;
    int integral = int.MaxValue;
    long longIntegral = long.MaxValue;

    float shortDecimal = float.MaxValue;
    double normalDecimal = double.MaxValue;
    decimal longDecimal = Decimal.MaxValue;

    char singleCharacter = 'A';

    char? nullableType = null;
    
    enum Days {
        FRIDAY, SATURDAY, SUNDAY
    }
    
    struct Point
    {
        public double X { get; }
        public double Y { get; }
    
        public Point(double x, double y) => (X, Y) = (x, y);
    }
}